src/modules/rtpengine/rtpengine.c

```diff
@@ -1442,9 +1443,9 @@ mod_init(void)
 	/* select the default set */
 	default_rtpp_set = select_rtpp_set(setid_default);
 	if (!default_rtpp_set) {
-		LM_NOTICE("Default rtpp set %d NOT found\n", setid_default);
+		LM_NOTICE("Default rtpp set %u NOT found\n", setid_default);
```

Comment:
The type of `setid_default` itself also ought to be changed to unsigned accordingly